src/styles/main.scss

```
@@ -0,0 +1,90 @@
+$center-size: 30px;
+$clock-size: 500px;
+
+$seconds-hand-width: 10px;
+$seconds-hand-heigh: 225px;
+
+$minute-hand-heigh: 150px;
+$minute-hand-width: 10px;
+
+$hand-border-radius: 10px;
+
+body {
+  margin: 0;
+  padding: 0;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  height: 100vh;
+}
+
+.clock {
+  height: $clock-size;
+  width: $clock-size;
+
+  border-radius: 50%;
+
+  position: relative;
+
+  background-color: beige;
+
+  &__center {
+    background-color: blue;
+
+    width: $center-size;
+    height: $center-size;
+
+    border-radius: 50%;
+
+    transform: translate(-35%, 50%);
+
+    z-index: 1;
+
+    position: absolute;
+    bottom: 50%;
+    left: 50%;
+  }
+
+  &__seconds-hand {
+    background-color: blue;
+    width: $seconds-hand-width;
+    height: $seconds-hand-heigh;
+
+    transform-origin: bottom center;
+
+    animation: round 60s steps(60) infinite;
+
+    border-radius: $hand-border-radius;
+  }
+
+  &__minute-hand {
+    background-color: red;
+    width: $minute-hand-width;
+    height: $minute-hand-heigh;
+
+    transform-origin: bottom center;
+
+    animation: round 3600s linear infinite;
+
+    border-radius: $hand-border-radius;
+  }
+
+  &__seconds-hand,
+  &__minute-hand {
+    position: absolute;
+    bottom: 50%;
+    left: 50%;
+  }
```

but better create mixin

```suggestion
  &__center,
  &__seconds-hand,
  &__minute-hand {
    position: absolute;
    bottom: 50%;
    left: 50%;
  }
```